FhirToDataLake/deploy/templates/DeployContainerApp.json

```diff
@@ -0,0 +1,353 @@
+                {
+                    "name": "job__startTime",
+                    "value": "[parameters('dataStart')]"
+                },
+                {
+                    "name": "job__endTime",
+                    "value": "[parameters('dataEnd')]"
+                },
+                {
+                    "name": "job__tableUrl",
+                    "value": "[concat('https://', variables('storageAccountName'), '.table.core.windows.net')]"
+                },
+                {
+                    "name": "job__queueUrl",
+                    "value": "[concat('https://', variables('storageAccountName'), '.queue.core.windows.net')]"
+                },
+                {
+                    "name": "dataLakeStore__storageUrl",
+                    "value": "[concat('https://', variables('storageAccountName'), '.blob.core.windows.net')]"
+                },
+                {
+                    "name": "filter__filterScope",
+                    "value": "[parameters('filterScope')]"
+                },
+                {
+                    "name": "filter__enableExternalFilter",
+                    "value": "[if(empty(variables('filterConfigImageReference')), 'false', 'true')]"
+                },
+                {
+                    "name": "filter__filterImageReference",
+                    "value": "[variables('filterConfigImageReference')]"
+                },
+                {
+                    "name": "filter__groupId",
+                    "value": "[parameters('groupId')]"
+                },
+                {
+                    "name": "filter__requiredTypes",
+                    "value": "[parameters('requiredTypes')]"
+                },
+                {
+                    "name": "filter__typeFilters",
+                    "value": "[parameters('typeFilters')]"
+                },
+                {
+                    "name": "schema__enableCustomizedSchema",
+                    "value": "[if(empty(parameters('customizedSchemaImageReference')), 'false', 'true')]"
+                },
+                {
+                    "name": "schema__schemaImageReference",
+                    "value": "[parameters('customizedSchemaImageReference')]"
+                },
+                {
+                    "name": "fhirServer__serverUrl",
+                    "value": "[parameters('fhirServerUrl')]"
+                },
+                {
+                    "name": "fhirServer__version",
+                    "value": "[parameters('fhirVersion')]"
+                },
+                {
+                    "name": "fhirServer__authentication",
+                    "value": "[if(parameters('authentication'), 'ManagedIdentity', 'None')]"
+                }
+              ],
+              "resources": {
+                "cpu": 2,
+                "memory": "4Gi"
+              }
+            }
+          ],
+          "scale": {
+            "minReplicas": 1,
+            "maxReplicas": 3
```

Comment:
can we use a parameter like maxInstanceCount with default value 5?

Reply:
Sure, updated.